### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by awaisdar001**
Adds different sections on the `ReadMe.md`. 
Adds new `CONTRIBUTING.md` file.

## CONTRIBUTING.md

Screen Shot 2021-08-12 at 10 29 09 PM

## ReadMe.md
Screen Shot 2021-08-12 at 10 29 25 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
 
-The search.html page is powered by cse.google.com, configured here:
-https://cse.google.com/cse/setup/basic?cx=003935656615667250026:jtodz7s8c9a
+The `search.html` page is powered by Google Programmable Search Engine and can be configured [here](https://cse.google.com/cse/setup/basic?cx=003935656615667250026:jtodz7s8c9a):
 
-The current list of sites that should be configured there is in search-sites.txt.
+The current list of sites that should be configured there is in `search-sites.txt`.
+
+## Contribution
+If you want to contibute, please open a pull request against master and tag [community-engineering](https://github.com/orgs/edx/teams/community-engineering) team for a review. There is no extra release process and as soon as you merge to master, the changes will be reflected staright away on production.
```
Comment (ormsbee) [inline README.md]: Can people who are not edX employees see that team information? Is there an email address they can reach out to?

Comment (nsprenkle) [reply README.md]: I don't think teams are public. At least hitting it when logged out of GitHub doesn't show up.

Comment (nsprenkle) [reply README.md]: That said, they are a codeowner so should be auto-added to PRs, yes?

Comment (awaisdar001) [reply README.md]: Good call, let me check. 
if they are not public, how we should reword it to indicate that they are the owners and should be tagged for the changes?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+## Search
 
-The search.html page is powered by cse.google.com, configured here:
-https://cse.google.com/cse/setup/basic?cx=003935656615667250026:jtodz7s8c9a
+The `search.html` page is powered by Google Programmable Search Engine and can be configured [here](https://cse.google.com/cse/setup/basic?cx=003935656615667250026:jtodz7s8c9a):
 
-The current list of sites that should be configured there is in search-sites.txt.
+The current list of sites that should be configured there is in `search-sites.txt`.
+
+## Contribution
```
Comment (nsprenkle) [inline README.md]: I like the idea, but this should probably be in `.github/CONTRIBUTORS.md` (see some other edX repos for examples). This is a more standard and GitHub-integrated way of surfacing that info.

Comment (awaisdar001) [reply README.md]: I think github talks about having that file at the root, not in `.github/` can you confirm? 
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

File: README.md (hunk 1 above)
Comment (awaisdar001) [reply README.md]: Yes, 404 for non-edx team member.

Comment (awaisdar001) [reply README.md]: I have removed the _team_ reference/link from the document now.

File: README.md (hunk 2 above)
Comment (nsprenkle) [reply README.md]: ^ The doc allows for that file in root, `docs`, or `.github`. `.github` is my personal preference but it could be in any of those 3.

Comment (awaisdar001) [reply README.md]: yes, right :)

**Comment by awaisdar001** [comment]
@nsprenkle if this is good to go, can you please 👍 ?

**Review by mehaknasir — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,8 +1,12 @@
...
 
-The search.html page is powered by cse.google.com, configured here:
-https://cse.google.com/cse/setup/basic?cx=003935656615667250026:jtodz7s8c9a
+The `search.html` page is powered by Google Programmable Search Engine and can be configured [here](https://cse.google.com/cse/setup/basic?cx=003935656615667250026:jtodz7s8c9a):
 
-The current list of sites that should be configured there is in search-sites.txt.
+The current list of sites that should be configured there is in `search-sites.txt`.
+
+## Release
+There is no extra release process and as soon as you merge to master, the changes will be reflected staright away on production.
```
Comment (nedbat) [inline README.md]:
```suggestion
There is no extra release process and as soon as you merge to master, the changes will be reflected straight away on production. 
```

File: README.md [hunk 4]
```diff
@@ -1,8 +1,12 @@
-This is the Github-hosted branch of edx-platform that is behind
-http://docs.edx.org
+## Overview
+This is the Github-hosted and tied to the master branch that is behind
```
Comment (nedbat) [inline README.md]:
```suggestion
This is the Github Pages-hosted repo behind
```

**Review by nedbat — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (awaisdar001) [reply README.md]: good catch.

**Comment by awaisdar001** [comment]
Feedback addressed - PR is updated.

**Review by leangseu-edx — APPROVED** [review]
👍

**Review by nedbat — APPROVED** [review]
